src/coreclr/jit/inlinepolicy.cpp

```diff
@@ -502,11 +502,25 @@ bool DefaultPolicy::BudgetCheck() const
         //
         assert(m_IsForceInlineKnown);
         assert(m_CallsiteDepth > 0);
-        const bool allowOverBudget = m_IsForceInline && (m_CallsiteDepth <= strategy->GetMaxForceInlineDepth());
+        bool allowOverBudget = m_IsForceInline && (m_CallsiteDepth <= strategy->GetMaxForceInlineDepth());
+
+        if (!allowOverBudget && (m_CodeSize <= (InlineStrategy::ALWAYS_INLINE_SIZE / 2)))
```

Comment:
`ALWAYS_INLINE_SIZE / 2` (8 bytes) is probably too conservative, but I was targeting simple properties

Reply:
Did you look at what happened for larger values?

Reply:
1 party web service stats:
Main: 2350 "exceed budget"
This PR: 2173 "exceed budget"
This PR + 16 bytes: 1865 "exceed budget"
This PR + 32 bytes: 1743 "exceed budget"
This PR + 64 bytes: 1743 "exceed budget" (same number)

Sounds like 16b is a good choice?

Reply:
Yep, it's 16

Reply:
Yes, sounds good.